packages/frameworks-solid-start/src/index.ts

```diff
@@ -22,17 +22,21 @@ const actions: AuthAction[] = [
   "error",
 ]
 
-// currently multiple cookies are not supported, so we keep the next-auth.pkce.code_verifier cookie for now:
+// currently multiple cookies are not supported, so we keep the important ones
```

Comment:
@OrJDev It seems that solid-start got the multiple cookie support yesterday: https://github.com/solidjs/solid-start/pull/656

Reply:
Awesome yes, i will update this pr today.

Reply:
https://github.com/nextauthjs/next-auth/pull/6248/commits/20bc4325ad33580902469068aaa43262a1797c3e